tests/envs/test_playroom.py

```diff
@@ -260,23 +262,76 @@ def test_playroom_options():
     utils.update_config({"env": "playroom"})
     env = PlayroomEnv()
     env.seed(123)
-    Pick = [o for o in env.options if o.name == "Pick"][0]
-    Stack = [o for o in env.options if o.name == "Stack"][0]
-    PutOnTable = [o for o in env.options if o.name == "PutOnTable"][0]
-    block_type = [t for t in env.types if t.name == "block"][0]
     robot_type = [t for t in env.types if t.name == "robot"][0]
+    block_type = [t for t in env.types if t.name == "block"][0]
+    door_type = [t for t in env.types if t.name == "door"][0]
+    dial_type = [t for t in env.types if t.name == "dial"][0]
+    On = [p for p in env.predicates if p.name == "On"][0]
+    OnTable = [p for p in env.predicates if p.name == "OnTable"][0]
+    Clear = [p for p in env.predicates if p.name == "Clear"][0]
+    LightOn = [p for p in env.predicates if p.name == "LightOn"][0]
+    robot = robot_type("robby")
     block0 = block_type("block0")
     block1 = block_type("block1")
-    robot = robot_type("robby")
+    block2 = block_type("block2")
+    door1 = door_type("door1")
+    door2 = door_type("door2")
+    door3 = door_type("door3")
+    door4 = door_type("door4")
+    door5 = door_type("door5")
+    door6 = door_type("door6")
+    dial = dial_type("dial")
     task = env.get_train_tasks()[0]
     state = task.init
-    # Right now this is just for code coverage -- to add more later
-    Pick.ground([robot, block0], np.array(
-        [0, -1, 0], dtype=np.float32)).policy(state)
-    Stack.ground([robot, block1], np.array(
-        [0, 0, 0], dtype=np.float32)).policy(state)
-    PutOnTable.ground([robot], np.array(
-        [0.5, 0.5], dtype=np.float32)).policy(state)
+    # Run through a specific plan of options.
+    Pick = [o for o in env.options if o.name == "Pick"][0]
+    Stack = [o for o in env.options if o.name == "Stack"][0]
+    PutOnTable = [o for o in env.options if o.name == "PutOnTable"][0]
+    Move = [o for o in env.options if o.name == "Move"][0]
+    OpenDoor = [o for o in env.options if o.name == "OpenDoor"][0]
+    CloseDoor = [o for o in env.options if o.name == "CloseDoor"][0]
+    TurnOnDial = [o for o in env.options if o.name == "TurnOnDial"][0]
+    TurnOffDial = [o for o in env.options if o.name == "TurnOffDial"][0]
+    option_sequence = [
+        Move.ground([robot], [2.0, 30.0, 0.0]),
+        Move.ground([robot], [2.0, 30.0, 0.0]),
```

Comment:
this option is here twice because the `option_policy` defined below seems to always "bypass" the first option (since terminal is always true). not sure if I should do something about this?

Reply:
good point -- the option_policy code that I referred you to is outdated. can you change both this one and the other unit test so that they instead use `utils.option_plan_to_policy`? then repeating the first option should not be necessary. thanks & sorry!